From: Roland Fehrenbacher <rf@q-leap.de>
Date: Sat, 18 Feb 2017 14:58:30 +0100
Subject: QL - Fix exception in job.get_job_ptr due to wrong string handling

---
 pyslurm/pyslurm.pyx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyslurm/pyslurm.pyx b/pyslurm/pyslurm.pyx
index def7338..bb40ba3 100644
--- a/pyslurm/pyslurm.pyx
+++ b/pyslurm/pyslurm.pyx
@@ -2160,7 +2160,7 @@ cdef class job:
             Job_dict[u'start_time'] = self._record.start_time
 
             if self._record.state_desc:
-                Job_dict[u'state_reason'] = self._record.state_desc.replace(" ", "_")
+                Job_dict[u'state_reason'] = self._record.state_desc.decode().replace(" ", "_")
             else:
                 Job_dict[u'state_reason'] = slurm.slurm_job_reason_string(
                     <slurm.job_state_reason>self._record.state_reason)
