From: Roland Fehrenbacher <rf@q-leap.de>
Date: Sun, 27 Aug 2017 16:39:41 +0200
Subject: QL - Bump Slurm support to 17.02.7

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 6e524bc..3404230 100644
--- a/setup.py
+++ b/setup.py
@@ -24,7 +24,7 @@ logging.basicConfig(level=20)
 #VERSION = imp.load_source("/tmp", "pyslurm/__init__.py").__version__
 __version__ = "17.02.0"
 __min_slurm_hex_version__ = "0x110200"
-__max_slurm_hex_version__ = "0x110206"
+__max_slurm_hex_version__ = "0x110207"
 
 def fatal(logstring, code=1):
     logger.error("Fatal: " + logstring)
