From: Roland Fehrenbacher <rf@q-leap.de>
Date: Wed, 13 Sep 2017 13:40:54 +0200
Subject: QL - Correct cpu_freq fields of jobdict

---
 pyslurm/pyslurm.pyx | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pyslurm/pyslurm.pyx b/pyslurm/pyslurm.pyx
index 122168e..880fafa 100644
--- a/pyslurm/pyslurm.pyx
+++ b/pyslurm/pyslurm.pyx
@@ -2055,12 +2055,12 @@ cdef class job:
             if self._record.cpu_freq_gov == slurm.NO_VAL:
                 Job_dict[u'cpu_freq_gov'] = None
             else:
-                Job_dict[u'cpu_freq_gov'] = self._record.cpu_freq_min
+                Job_dict[u'cpu_freq_gov'] = self._record.cpu_freq_gov
 
             if self._record.cpu_freq_max == slurm.NO_VAL:
                 Job_dict[u'cpu_freq_max'] = None
             else:
-                Job_dict[u'cpu_freq_max'] = self._record.cpu_freq_min
+                Job_dict[u'cpu_freq_max'] = self._record.cpu_freq_max
 
             if self._record.cpu_freq_min == slurm.NO_VAL:
                 Job_dict[u'cpu_freq_min'] = None
